findbugs-exclude.xml

```diff
@@ -0,0 +1,5 @@
+<FindBugsFilter>
+     <Match>
+       <Class name="~.+\.Messages" />
```

Comment:
I added it because I was getting Findbugs errors because of the generated `Messages` class. I will remove it and investigate further. Thanks for the tip.